interface/modules/custom_modules/oe-module-weno/templates/indexrx.php

```diff
@@ -188,16 +219,32 @@
             </div>
         </div>
         <div class="container-xl mt-3">
-            <iframe id="wenoIfram"
-                title="Weno IFRAME"
-                width="100%"
-                height="900"
-                src="<?php echo $urlOut; ?>">
-            </iframe>
+            <iframe id="wenoIfram" title="Weno IFRAME" width="100%" height="900" src="<?php echo $urlOut; ?>"></iframe>
```

Comment:
Pretty sure $urlOut will need to be escaped with attr() to prevent cross scripting attack via the $provider_info['email'] variable.